src/lib/types.ts

```diff
@@ -7,6 +7,7 @@ export type Chapter = {
   }
   baseId: string
   acceptsSignups: boolean
```

Comment:
This is maybe not the most future-proof interface for managing signup state of chapters.

How about we use `acceptsSignups: Record<'students' | 'pupils', boolean> = {students: true, pupils: false}` ?